packages/legend-studio/src/stores/sidebar-state/WorkspaceReviewState.ts

```diff
@@ -262,6 +267,15 @@ export class WorkspaceReviewState {
   ): GeneratorFn<void> {
     this.isCreatingWorkspaceReview = true;
     try {
+      this.editorStore.projectConfigurationEditorState.originalProjectConfiguration?.projectDependencies.forEach(
```

Comment:
Could we add a small comment here (😭) to explain that SDLC will catch this and when we merge the review? Thanks!

Reply:
you can add the comment yourself since you did not want  to do this via legend studio thanks

Reply:
![image](https://user-images.githubusercontent.com/13574879/176294840-3575cf8b-b9fa-4a69-89c0-0a7b0ec43aca.png)